package.json

```
@@ -4,25 +4,27 @@
   "description": "Superset UI Plugins",
   "private": true,
   "scripts": {
-    "build": "yarn build:cjs && yarn build:esm && yarn run type:dts && yarn build:assets",
-    "build:cjs": "NODE_ENV=production beemo babel --extensions=\".js,.jsx,.ts,.tsx\" ./src --out-dir lib/ --delete-dir-on-start --minify --workspaces=\"@superset-ui/!(plugins-demo)\"",
-    "build:esm": "NODE_ENV=production beemo babel --extensions=\".js,.jsx,.ts,.tsx\" ./src --out-dir esm/ --delete-dir-on-start --esm --minify --workspaces=\"@superset-ui/!(plugins-demo)\"",
-    "build:assets": "node ./buildAssets.js",
+    "build": "yarn babel && yarn type:dts && yarn build:assets",
+    "babel": "yarn babel:cjs && yarn babel:esm",
+    "babel:cjs": "nimbus babel --clean --workspaces=\"@superset-ui/!(plugins-demo)\"",
+    "babel:esm": "nimbus babel --clean --workspaces=\"@superset-ui/!(plugins-demo)\" --esm",
+    "build:assets": "node ./scripts/buildAssets.js",
+    "clean": "rm -rf ./packages/**/{lib,esm}",
     "commit": "superset-commit",
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s -r 0",
-    "type": "NODE_ENV=production beemo typescript --workspaces=\"@superset-ui/((preset|plugin)-*)\" --noEmit",
-    "type:dts": "NODE_ENV=production beemo typescript --workspaces=\"@superset-ui/((preset|plugin)-*)\" --emitDeclarationOnly",
-    "lint": "beemo create-config prettier && beemo eslint \"./packages/*/{src,test,storybook}/**/*.{js,jsx,ts,tsx}\"",
-    "lint:fix": "beemo create-config prettier && beemo eslint --fix \"./packages/*/{src,test,storybook}/**/*.{js,jsx,ts,tsx}\"",
-    "jest": "beemo jest --color --coverage --react",
-    "postrelease": "lerna run deploy-demo",
-    "prepare-release": "git checkout master && git pull --rebase origin master && lerna bootstrap && yarn run test",
-    "prerelease": "yarn run build",
-    "pretest": "yarn run lint",
-    "prettier": "beemo prettier \"./packages/*/{src,test,storybook}/**/*.{js,jsx,ts,tsx,json,md}\"",
-    "release": "yarn run prepare-release && lerna publish && yarn run postrelease",
-    "test": "yarn run type && yarn run jest",
-    "test:watch": "yarn run lint:fix && beemo create-config jest --react && jest --watch"
+    "format": "yarn prettier --write",
+    "jest": "NODE_ENV=test nimbus jest --coverage --verbose",
+    "lint": "nimbus eslint",
```

If this is the task run by CI, should we validate pretty-ness here, e.g.:
```suggestion
    "lint": "nimbus eslint && nimbus prettier --check",
```

---

I think the commit hook with `husky` should auto format, but doesn't hurt to double check I guess.